src/ViewInfo.h

```diff
@@ -99,15 +99,18 @@ class AUDACITY_DLL_API NotifyingSelectedRegion : public wxEvtHandler
 };
 
 // See big pictorial comment in TrackPanel.cpp for explanation of these numbers
+//! constants related to y coordinates in the track panel
 enum : int {
-   // constants related to y coordinates in the track panel
+   kVerticalPadding = 7, /*!< Width of padding below each channel group */
    kAffordancesAreaHeight = 20,
    kTopInset = 4,
    kTopMargin = kTopInset + kBorderThickness,
    kBottomMargin = kShadowThickness + kBorderThickness,
    kSeparatorThickness = kBottomMargin + kTopMargin,
 };
 
+static_assert( kVerticalPadding == kTopMargin + kBottomMargin );
```

Comment:
Why not define this constant like this in enum?
`kVerticalPadding = kTopMargin + kBottomMargin`

Reply:
I explain in a later commit.  I want to move constants into a different enum.  I want to lot the constant kVerticalPadding be a more widely available constant but kTopMargin and kBottomMargin are meant to be exposed in fewer places.

So kVerticalPadding is not defined publicly in terms of them.  The consistency constraint on the three values becomes a static_assert instead.